docker/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# Dockerfile for awesomeWM.
```

Comment:
Could you move the Dockerfile into a subdirectory? At some point we might/should have many Dockerfiles for jobs like a PPA upload and older distribution support (executed on a timely base instead of each PR changes).